srcpkgs/intel-parallel-studio-xe/files/Makefile-dkms

```diff
@@ -0,0 +1,132 @@
+#
```

Comment:
This preferably will be downloaded with distfiles too.

Reply:
This is a hand-made file to appease the DKMS way of building kernel modules.